src/operator/softmax_output-inl.h

```diff
@@ -37,27 +37,28 @@ struct SoftmaxOutputParam : public dmlc::Parameter<SoftmaxOutputParam> {
   bool out_grad;
   DMLC_DECLARE_PARAMETER(SoftmaxOutputParam) {
     DMLC_DECLARE_FIELD(grad_scale).set_default(1.0f)
-    .describe("Scale the gradient by a float factor");
+    .describe("Scale the gradient by multiplying a float factor.");
     DMLC_DECLARE_FIELD(ignore_label).set_default(-1.0f)
-    .describe("the labels with value equals to ``ignore_label`` will be ignored "
-              "during backward (only works if "
-              "use_ignore is set to be true).");
+    .describe("The instances whose ``labels == {ignore_label}``will be ignored "
+              "during backward. (if and only if ``use_ignore`` is set to be ``true``).");
     DMLC_DECLARE_FIELD(multi_output).set_default(false)
-    .describe("If set to true, softmax will applied on axis 1");
+    .describe("If this is set to ``true``, the softmax function will be computed along with "
+              "the second axis.");
     DMLC_DECLARE_FIELD(use_ignore).set_default(false)
-    .describe("If set to true, the ignore_label value will not contribute "
-      "to the backward gradient");
+    .describe("If this is set to ``true``, the ``ignore_label`` value will not contribute "
```

Comment:
ignore_label should be in single backticks because it is a variable/parameter name